### DISCUSSION TITLE
Terratest get kubernetes logs

### DISCUSSION THREAD

**Discussion by tomikonio** [gruntwork-io/knowledge-base#596] [How do I]
Hi,
I have a k8s job that I test against.
I want to print the job's logs in case of failure.
How can I get the logs?

---

<ins datetime="2022-11-23T07:51:16Z">
  <p><a href="https://support.gruntwork.io/hc/requests/109638">Tracked in ticket #109638</a></p>
</ins>

**Comment by denis256** [comment]
Hi,
AFAIK it is not supported from test functions, however can be used `kubectl logs <pod-name>` to read pod logs
